themes/classic/templates/checkout/_partials/steps/payment.tpl

```diff
@@ -30,7 +30,7 @@
                 name="payment-option"
                 type="radio"
                 required
-                {if $selected_payment_option == $option.id || $is_free} checked {/if}
+                {if ($selected_payment_option == $option.id || $is_free) || $payment_options|@count === 1} checked {/if}
```

Comment:
@NeOMakinG do we know what happens when we have more `$module_options`? I mean we have one payment method, but with many module_options? this will fail right?

Reply:
@Matt75 could be helpful here, he's been working on `paymentexample` module and know this topic best I think :)

Reply:
Oh well I'm stupid, of course it will fail, we need to check `module_options` too..

Reply:
It should be fine now @kpodemski : It's verifying that $module_options has only one item too